ci/travis-after-success.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 set -xe
 
+test "$AFTER_SCRIPT" = "1" || exit 0
```

Comment:
Mmm, this will make the script exit with a non 0 status if the test fail, because `-e` will make the shell exit as soon as the `test` command fails, so `exit 0` is never executed. At least in theory, but it seems like tests passed, I don't know why.